whats_fresh/whats_fresh_api/templates/story.html

```diff
@@ -39,18 +39,18 @@ <h4 id='error_tag' class='light warning'></h4>
         <h3>Basic Info</h3>
         <p class="field_text">Name*:</p>
         {{ story_form.name }}
-        <p class="field_text">Facts:</p>
+        <p class="field_text">Overview:</p>
```

Comment:
Ah, I see, Github just decided to pull in this commit for whatever reason. Never mind!